tests/phpunit/ConfigTest.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\SnowflakeTransformation\Tests;
+
+use Keboola\SnowflakeTransformation\Config;
+use Keboola\SnowflakeTransformation\ConfigDefinition;
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Config\Definition\Exception\InvalidConfigurationException;
+
+class ConfigTest extends TestCase
+{
+    public function testConfig(): void
+    {
+        $configArray = [
+            'authorization' => $this->getDatabaseConfig(),
+            'parameters' => [
+                'steps' => [
+                    [
+                        'name' => 'first step',
+                        'blocks' => [
+                            [
+                                'name' => 'first block',
+                                'script' => [
+                                    'DROP TABLE IF EXISTS "output"',
+                                ],
+                            ],
+                        ],
+                    ],
+                ],
+                'query_timeout' => '7200',
```

Comment:
jeste by to chtelo test na to, co se stane kdyz tam neni number a kdyz tam neni vubec nic (ze se tam hodi default) (ten default bude dost dulezitej)

Reply:
přepsal jsem query_timeout na `integerNode` + test na chybnou hodnotu